client/src/store/new-request-module.ts

```diff
@@ -1356,10 +1348,37 @@ export class NewRequestModule extends VuexModule {
           data = this.editNameReservation
           break
       }
-      if (this.showCorpNum && this.corpNum) {
+
+      // TODO: Should be able to remove this...
+      /* if (this.showCorpNum && this.corpNum) {
         data['corpNum'] = this.corpNum
+      } */
+
+      const response = await axios.put(`/namerequests/${nrNum}`, data, {
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      })
+
+      this.setNrResponse(response.data)
+
+      const { nr } = this
+      const { applicants = [] } = nr
+
+      if (applicants instanceof Array) {
+        this.setApplicantDetails(applicants[0])
+      } else if (applicants) {
+        this.setApplicantDetails(applicants)
       }
-      response = await axios.put(`/namerequests/${nrNum}`, data, {
+    } catch (error) {
+      // eslint-disable-next-line
+      console.log(error)
+    }
+  }
+  @Action
```

Comment:
Only add these chnages

Reply:
Bad merge, and a couple differences with my auto-format code style cause i'm using webstorm not vscode. Resolved.